web_external/ImagesGallery/ImagesFilters.js

```diff
@@ -131,10 +137,14 @@ _.extend(isic.collections.ImagesFilters.prototype, Backbone.Events, {
                     ')';
                 return '(' + lowBoundExpression + ' or ' + highBoundExpression + ')';
             })
-            .value()
-            // Combine all expressions
-            .join(' and ');
-        return '(' + filterExpression + ')';
+            .value();
+        if (facetFilters['null'] === false) {
```

Comment:
What's the idea in this conditional block? To filter out all null values if the caller asks for that?